pkg/k8sselector/selection/operator.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2016 The Kubernetes Authors.
```

Comment:
Since we will probably have more k8s-related logic would it make sense to have a `pkg/k8s/{selector,whatever}`